package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "capra_web_ui",
-  "version": "2.4.7",
+  "version": "2.4.9",
```

Comment:
What happened to .8? 👀

Reply:
It was a pre release based on this branch during the Robocup so the version never changed on master